nuget/Microsoft.Windows.CsWinRT.Embedded.targets

```diff
@@ -10,25 +10,59 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 <!-- Activate the internal version of the runtime layer --> 
     <DefineConstants>$(DefineConstants);EMBED</DefineConstants>
 <!-- Path to runtime sources, must update if their location changes in the .nuspec --> 
-    <CsWinRTEmbeddedSources>$(CsWinRTPath)contentFiles\cs\any\</CsWinRTEmbeddedSources>
-
+    <CsWinRTEmbeddedTFMNet5OrGreater Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) >= 5">true</CsWinRTEmbeddedTFMNet5OrGreater>
   </PropertyGroup>
 
-<!-- we have to manually include the files from any\, in contrast to how 
-     files under a TFM specific folder (e.g. net5.0\) get brought in automatically     --> 
   <ItemGroup>
-
-    <Compile Include="$(CsWinRTEmbeddedSources)**\*.cs" />
-
+    <CsWinRTEmbeddedSourcesAny Include="$(CsWinRTPath)embedded\any\*.cs"/>
+    <CsWinRTEmbeddedSourcesNet5 Include="$(CsWinRTPath)embedded\net5.0\*.cs"/>
+    <CsWinRTEmbeddedSourcesNet2 Include="$(CsWinRTPath)embedded\netstandard2.0\*.cs"/>
   </ItemGroup>
-
+ 
+  <!-- Copy sources that are for any framework -->
+  <Target Name="CsWinRTAddEmbeddedRuntime_AnySources"
+          AfterTargets="CsWinRTGenerateProjection">
```

Comment:
Do you also need a BeforeTargets="CsWinRTIncludeProjection"

Reply:
Good idea to add it, we've seen MSBuild issues come in before because AfterTargets wasn't enough in certain cases